cfme/fixtures/pytest_selenium.py

```diff
@@ -53,6 +58,11 @@
 urls = []
 
 
+removed_module(__name__, removal_version="framework 3.0")
```

Comment:
we might want to declare some smaller wrappers in something like utils.deprecation